docs/pages/application-access/guides/dynamodb.mdx

```diff
@@ -1,22 +1,30 @@
 ---
-title: Database Access with AWS DynamoDB
-description: How to access AWS DynamoDB with Teleport.
+title: AWS DynamoDB using the Teleport Application Service
+description: How to access AWS DynamoDB through the Teleport Application Service
 ---
 
-Access to AWS DynamoDB is provided by [**Teleport Application
+Access to AWS DynamoDB can be provided by [**Teleport Application
 Access**](../../application-access/introduction.mdx) for the AWS Console and
-API.
+API. This is an alternative to accessing DynamoDB through the Teleport Database
+service, as described in our [Database Access with AWS
+DynamoDB](../../database-access/guides/aws-dynamodb.mdx) guide.
+
+Note that using the Database Service will allow you to connect with GUI clients,
+whereas the Application Service does not. On the other hand, a single Application
+Service node can access DynamoDB across regions, while Database nodes must be
+present in the same region(s) as the DynamoDB database(s).
```

Comment:
Avoid using "node" as that term is vague. I would rephrase this:

```suggestion
Note that using the Database Service will allow you to connect with GUI clients,
whereas the Application Service does not. On the other hand, a single Teleport application
can be configured to access DynamoDB in any AWS region, while a Teleport Database must be
configured to access DynamoDB in each AWS region.
```

Reply:
And actually I just thought of another one: Database access supports per-session-MFA and hardware backed keys, while application access does not (currently).

Reply:
I'm ok with not using "node", but the suggested edit still needs some noun, otherwise "application" and "Database" becomes the noun. Since, in the latter case for example, it's not an actual database that teleport runs. How about:


```suggestion
Note that using the Database Service will allow you to connect with GUI clients,
whereas the Application Service does not. On the other hand, a single Application
Service instance can access DynamoDB across regions, while Teleport Database
instances must be present in the same region(s) as the DynamoDB database(s).
```

Reply:
how about "resource" instead of "instance"? I just don't the wording here to sound like you need an agent running in the same AWS region as the dynamodb endpoint.

Reply:
OK, I think I misunderstood [this earlier comment](https://github.com/gravitational/teleport/pull/19956#discussion_r1086015392):

> one advantage is you do not need to configure app service dynamodb for each region you intend to access

I _thought_ this implied that database access did require a service in each region. Can we clarify what this means? I tagged you for review of that change in that thread as well.

Reply:
"service" refers to the agent in my opinion. It's the `db_service` section of the yaml.

You do not need multiple agents (services). You need one configured database per region. They can all be configured on one agent.

Reply:
Clarified in https://github.com/gravitational/teleport/pull/19956/commits/f64a9fa3a51b5bbee592d177afe76a9e1799060a